### PR TITLE
Az.IoTCentral - Allow updating application pricing information using Set-AzIotCentralApp command

### PR DESCRIPTION
**Pull request by r-delgadillo**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs [hunk 1]
```diff
@@ -46,6 +46,14 @@ public class SetAzureRmIotCentralApp : IotCentralFullParameterSetCmdlet
         [ValidateNotNullOrEmpty]
         public Hashtable Tag { get; set; }
 
+        [Parameter(
```
Comment (poanchen) [inline src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs:49]: these are generated by autotest correct?

File: src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md [hunk 2]
```diff
@@ -83,6 +84,29 @@ Template          : iotc-default@1.0.0
...
+                    .IoTCentral/IoTApps/MyAppResourceName
+Name              : MyAppResourceName
+Type              : Microsoft.IoTCentral/IoTApps
+Location          : westus
+Tag               : 
+Sku               : Microsoft.Azure.Commands.IotCentral.Models.PSIotCentralAppSkuInfo
+ApplicationId     : XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+DisplayName       : Display Name
+Subdomain         : MyAppSubdomain
+Template          : iotc-default@1.0.0
```
Comment (poanchen) [inline src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md:106]: should we use iotc-pnp-preview instead?

File: src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs (hunk 1 above)
Comment (r-delgadillo) [reply src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs:49]: Not that I'm aware of. I don't THINK powershell uses autorest, only arm swagger file. Did I miss something?

Comment (poanchen) [reply src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs:49]: Just checked. Yea, I think you are right. We need to do it ourselves!

File: src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md (hunk 2 above)
Comment (r-delgadillo) [reply src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md:106]: Let's leave it as is for this PR. We can go through the documentation later.

Comment (poanchen) [reply src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md:106]: sounds good!

File: src/IotCentral/IotCentral/ChangeLog.md [hunk 3]
```diff
@@ -19,12 +19,13 @@
 -->
 ## Upcoming Release
 
-## Version 0.7.4
+## Version 0.7.5
+* Allow pricing information to be updated using Set-AzIoTCentralApp command.
```
Comment (isra-fel) [inline src/IotCentral/IotCentral/ChangeLog.md:23]: Please put the change log of the upcoming release under `## Upcoming Release`. Do not add new versions.

Check the top of this file for an exmaple

File: src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs (hunk 1 above)
Comment (isra-fel) [reply src/IotCentral/IotCentral/IotCentral/SetAzureRmIotCentralApp.cs:49]: We are using autorest to generate a few modules such as "Az.Databricks", but most of them are still hand written.

File: src/IotCentral/IotCentral/ChangeLog.md (hunk 3 above)
Comment (r-delgadillo) [reply src/IotCentral/IotCentral/ChangeLog.md:21]: @isra-fel  Oops,  fixed.

**Pull request merged**